### DISCUSSION TITLE
1.0.0

### DISCUSSION THREAD

**Discussion by kylo252** [LunarVim/LunarVim#2161] [Announcements]
###  Features

- _(info)_ Display overridden servers for filetype (#2155)
- _(installer)_ Nicer rsync output (#2067)
- _(luadev)_ Better vim api completion (#2043), thanks @meijieru!
- _(terminal)_ Lazygit can now be toggled (#2039)
- Auto resize for more consistent UI behavior
- Support new null-ls (#1955)
- Empty for empty buffers instead of Buffer <#>
- Improved LSP grouping in lualine
- Decrease hide in width limit for lualine
- Add support for fsharp (#2021)
- Add some messages in uninstall.sh (#1945), thanks @Spaxly!
- Null-ls code_actions interface (#2008), thanks @meijieru!
- Full compatibility with neovim v0.6 (#2037)
- Multiple enhancements to lvim-reload (#2054)
- Update lsp-installer and lspconfig hashes to enable solidity_ls language server (#2072), thanks @edag94!
- Use Telescope's git_files with fallback (#2089), thanks @LandonSchropp!
- Lazyload notify's configuration (#1855)

###  Bugfix

- _(autopairs)_ Add missing configuration entries (#2030)
- _(bootstrap)_ Remove hard-coded spellfile option (#2061)
- _(ci)_ Blacklist 'sorbet' (#1936)
- _(cmp)_ If possible jump in the snippet after completion
- _(cmp)_ Revert broken sequential loading (#2002)
- _(gitsigns)_ Rounded border (#2142), thanks @meijieru!
- _(install)_ Avoid data-races for `on_packer_complete` (#2157)
- _(installer)_ Better handling of existing files (#2066)
- _(installer)_ Backup linked files with rsync (#2081), thanks @lmarlow!
- _(installer)_ Check if npm-prefix is writable (#2091)
- _(installer)_ More robust yarn validation (#2113), thanks @fbaltor!
- _(keymap)_ Unset on reload, load correct order, add unset default (#1942)
- _(lsp)_ Correct map and prevent highlight leak of diagnostic signs in 0.6 (#1934)
- _(lsp)_ Avoid installing an overridden server (#1981)
- _(lsp)_ Prevent repeated setup call (#2048)
- _(lsp)_ Correct client_id parsing in lvim-info (#2071)
- _(lsp)_ Allow overriding servers with custom providers (#2070)
- _(lsp)_ Set the handlers opts for v0.6 as well (#2109)
- _(lsp)_ Formatter now use new null-ls api function (#2135), thanks @cpea2506!
- _(lsp)_ No need to stop clients on LvimReload (#2160)
- _(lualine)_ Change `fg` of section `a` in onedarker (#1909), thanks @younger-1!
- _(null-ls)_ Allow the same linter and formatter (#1968)
- _(null-ls)_ Avoid sending invalid opts.args (#2154)
- _(nvimtree)_ Update settings (#2001)
- _(nvimtree)_ Restore default mappings + make them customizable (#2007), thanks @xeluxee!
- _(nvimtree)_ Handle paths containing spaces (#2027), thanks @xeluxee!
- _(plugins)_ Typo of pin commit of `treesitter` (#2046), thank @younger-1!
- _(terminal)_ Allow disabling the open binding for toggleterm
- _(which-key)_ Temporary solution for which-key (#2150), thanks @meijieru!
- _(windows)_ Autocmd requires forward slashes (#1967)
- _(windows)_ Remove redundant `resolve` call (#1974)
- Update the uninstallation script (#1924)
- Allow LunarVim changelog to work outside the lvim directory (#1952), thanks @lmarlow!
- Use an indepdent shadafile from neovim (#1910)
- Packersync issue when you have large number of plugins (#1922)
- No idea why this breaks barbar
- fix example of unmapping an lvim default (#1949), thanks @linsomniac!
- Lsp root can get very annoying when working with multiple languages. User is still able to turn it on.
- Update jdtls script
- Correct order for cmp's setup (#1999)
- Dont close if next char is a close pair and no pairs in same line (#2017)
- More accessible changelog (#2019)
- Better default, ignore `.git` in `live_grep` (#2020), thanks @meijieru!
- No restart required when changing colorscheme (#2026)
- No longer treat lazygit missing as an error (#2051)
- Remove autopairs cmp completion (#2083), thanks @LandonSchropp!
- Remove "error" message from git tag (#2141), thanks @ryaminal!

###  Refactor

- _(bootstrap)_ More robust git module (#2127)
- _(info)_ Use new null-ls api for sources (#2125)
- _(install.sh)_ Fix typo in node error message (#2107), thanks @stonepreston!
- _(null-ls)_ Allow passing full list of options for sources (#2137)
- _(settings)_ Add headless-mode settings (#2134)
- _(settings)_ Add headless-mode settings (#2134)
- _(term)_ Leave the first few ids unassigned (#2156)
- _(test)_ Cleanup test utilities (#2132)
- More configurable format-on-save (#1937)
- Load the default keymaps once (#1965)
- Remove unused old language configs (#2094)
- Uplift neovim's minimum version requirement to 0.6.0 (#2093)
- Avoid running ts.setup in headless (#2119)
- More consistent autocmds (#2133)
- Use a static lvim binary template (#1444)

**Full Changelog**: https://github.com/LunarVim/LunarVim/compare/0.6.1...1.0.0

---

As always, thanks a lot for all the wonderful contributors and the amazing community that we have! 

Sorry if I missed anyone.

<hr /><em>This discussion was created from the release <a href='https://github.com/LunarVim/LunarVim/releases/tag/1.0.0'>1.0.0</a>.</em>

**Comment by LandonSchropp** [comment]
Thank you all so much for the hard work on this project. I use LunarVim every day and I'm loving it!
